src/components/Table/__tests__/TableRow.spec.js

```diff
@@ -31,7 +31,7 @@ describe('TableRow', () => {
 
     const cell = queryByRole('cell');
     expect(cell).toBeInTheDocument();
-    expect(cell.outerHTML).toEqual('<td colspan=\"100%\">Hello Full Line Item</td>');
+    expect(cell.outerHTML).toEqual('<td colspan=\"100%\">HelloFull Line Item</td>');
```

Comment:
This test was failing and you can see the space on `'Hello '` seems to be getting trimmed off somehow.  I have no idea why this behaved differently before but this is what it's doing now. 🤷